src/controllers/controllermanager.cpp

```diff
@@ -218,24 +218,24 @@ void ControllerManager::slotSetUpDevices() {
         }
 
         // The filename for this device name.
-        QString presetBaseName = sanitizeString(name);
+        QString deviceName = sanitizeString(name);
 
-        // The first unique filename for this device (appends numbers at the end
-        // if we have already seen a controller by this name on this run of
-        // Mixxx.
-        presetBaseName = firstAvailableFilename(filenames, presetBaseName);
-
-        ControllerPresetPointer pPreset =
-                ControllerPresetFileHandler::loadPreset(
-                    presetBaseName + pController->presetExtension(),
-                    getPresetPaths(m_pConfig));
+        if (m_pConfig->getValueString(ConfigKey("[Controller]", deviceName)) != "1") {
```

Comment:
Is this the state from the enabled checkbox? If so, please add a comment. It is not obvious from the code.

Reply:
Yes it is. I'll add a comment.